### PR TITLE
Add data protection doc

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

File: using/legal/data-protection-policy.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Data Protection Policy
+
+## Overview
+
+The GDPR (General Data Protection Regulation) is UK legislation that governs how we process data originating from users from the UK.
+Exercism aims to be GDPR compliant for all users, even those outside the EU and you therefore have some important rights applicable to our Platform.
```
Comment (ErikSchierboom) [inline using/legal/data-protection-policy.md]: Isn't GDPR used across the EU: https://en.wikipedia.org/wiki/General_Data_Protection_Regulation?

File: using/legal/data-protection-policy.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+   Please contact privacy@exercism.org for more information.
+4. **Right to Erasure:**
+   We can remove your personal data from our systems so that you can no longer be personally identified to your solutions.
+   See the section below for details on how to remove your account.
+
+It is important to note that any content you upload to the site (including, but not limited to, solutions to exercises, comments, and mentor feedback) is transferred to us under a non-revocable license and therefore cannot be deleted from the Platform.
+
+## Deleting your data
+
+To delete your data, please log into your account, visit the [Settings page](https://exercism.org/settings) and click "Delete Account".
```
Comment (ErikSchierboom) [inline using/legal/data-protection-policy.md]: You probably also need to confirm deletion?

File: using/legal/README.md [hunk 3]
```diff
@@ -5,5 +5,6 @@ The following pages contain important information that explains your rights and
 - [Accessibility Statement](/docs/using/legal/accessibility)
 - [Code of Conduct](/docs/using/legal/code-of-conduct)
 - [Cookie Policy](/docs/using/legal/cookie-policy)
+- [Data Protection Policy (GDPR)](/docs/using/data-protection-policy)
```
Comment (ErikSchierboom) [inline using/legal/README.md]:
```suggestion
- [Data Protection Policy (GDPR)](/docs/using/legal/data-protection-policy)
```

File: using/legal/data-protection-policy.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+### Deleting data via email
+
+We take the risk of unintential data loss very seriously, and actively protect against malicious attempts to delete users data through email impersonation.
+The ICO states that organisations "have a legal responsibility to identify that an individual has made a request to you and handle it accordingly" and "if you have doubts about the identity of the person making the request you can ask for more information".
```
Comment (SaschaMann) [inline using/legal/data-protection-policy.md]: What's the ICO?

File: using/legal/data-protection-policy.md (hunk 1 above)
Comment (iHiD) [reply using/legal/data-protection-policy.md]: We're not part of the EU any more, so we just copied all the laws and are now governed by the UK ones. GDPR isn't actually a law though, even in the EU. It's a framework that each country interprets and creates their own law on.

File: using/legal/data-protection-policy.md (hunk 4 above)
Comment (iHiD) [reply using/legal/data-protection-policy.md]: https://ico.org.uk/

I'll expand the acronym - thanks.

File: using/legal/data-protection-policy.md (hunk 1 above)
Comment (ErikSchierboom) [reply using/legal/data-protection-policy.md]: Okay. Thanks for explaining

Comment (ErikSchierboom) [inline using/legal/data-protection-policy.md]:
```suggestion
Exercism aims to be GDPR compliant for all users, even those outside the UK, and you therefore have some important rights applicable to our Platform.
```

**Review by ErikSchierboom — APPROVED** [review]

File: using/legal/data-protection-policy.md (hunk 1 above)
Comment (SaschaMann) [reply using/legal/data-protection-policy.md]: > We're not part of the EU any more, so we just copied all the laws and are now governed by the UK ones. GDPR isn't actually a law though, even in the EU. It's a framework that each country interprets and creates their own law on.

This is not true. _Regulations_ are like laws, they immediately become enforceable in all member states. _Directives_ are more like frameworks that need to be interpreted and turned into national law.

**Pull request merged**